panel/theme/fast.py

```diff
@@ -170,10 +170,10 @@ class FastWrapper(ReactiveHTML):
 DEFAULT_STYLE = FastStyle()
 
 DARK_STYLE = FastStyle(
-    background_color="#181818",
+    background_color="#181818", #242424
```

Comment:
The Background and luminance changes the fundamental background colors to the original from 0.14.4

![image](https://user-images.githubusercontent.com/42288570/236109105-48e6bcb6-637b-4675-9993-a22c202609ba.png)

I can see on the `Fast` homepage they don't recommend using #181818 anymore. They recommend using #242424 or others. If I get the time before the 1.0 release I would like to 1) Create a FastStyler widget that can interactively change the basic design settings of a Fast template 2) Experiment with and determine the best settings.

But I believe this change improves the style a lot and is a safe bet.

Reply:
So which is it, should we switch to #242424? I think it looks better.

Reply:
Hmm, actually I don't think that looks good. Setting #242424 makes it very hard to distinguish the background and the card.